crates/bitcoin-da/src/service.rs

```diff
@@ -569,7 +569,7 @@ impl BitcoinService {
     pub async fn get_fee_rate_as_sat_vb(&self) -> Result<u64, anyhow::Error> {
         // If network is regtest or signet, mempool space is not available
         let smart_fee = match get_fee_rate_from_mempool_space(self.network).await {
-            Ok(fee_rate) => Some(fee_rate),
+            Ok(fee_rate) => fee_rate,
             Err(e) => {
                 tracing::warn!(?e, "Failed to get fee rate from mempool.space");
```

Comment:
With this change, it might be more appropriate to log error here maybe? Not exactly sure.

Reply:
Probably yes, you are right